lib/ui/painting/multi_frame_codec.cc

```diff
@@ -115,17 +115,17 @@ sk_sp<DlImage> MultiFrameCodec::State::GetNextFrameImage(
           << "Frame " << nextFrameIndex_ << " depends on frame "
           << requiredFrameIndex
           << " and no required frames are cached. Using blank slate instead.";
+      return nullptr;
     } else if (lastRequiredFrameIndex_ != requiredFrameIndex) {
       FML_DLOG(INFO) << "Required frame " << requiredFrameIndex
                      << " is not cached. Using blank slate instead.";
```

Comment:
So this path ends up getting hit for the problem GIF? It would seem like we're possibly drawing the wrong frame here, and we're just getting lucky that the frame still looks OK while rendering the wrong frame?

It's common for GIFs/WEBPs/APNGs to keep using one frame as the backdrop across the whole animation. That's not really possible with the way we've written multiframecodec today.

Not sure if we should fix it in this PR, but this comment should at least be updated since it's inaccurate with this change. Maybe just drop the "Using blank slate instead" since that doesn't always happen.

Reply:
It doesn't get hit becaues of the `else if`.

Reply:
I'mg oing to remove it entirely though.